bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -553,7 +553,7 @@ impl<'a> CircuitInputStateRef<'a> {
             sender_balance
         );
         // If receiver doesn't exist, create it
-        if (!receiver_exists && !value.is_zero()) || must_create {
+        if !receiver_exists && (!value.is_zero() || must_create) {
```

Comment:
We have a very similar logic in `transfer_to`:
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/ee97ab1f957918defafb9bc38dd51adf222ddfec/bus-mapping/src/circuit_input_builder/input_state_ref.rs#L630
Should we change that as well?

Reply:
`transfer_to` is only ever being called with `must_create` being false, so I just removed the argument.